### PR TITLE
[vcpkg update] List packages for vcpkg install

### PR DESCRIPTION
**Pull request by OlafvdSpek**

### CODE REVIEW

File: toolsrc/src/commands_update.cpp [hunk 1]
```diff
@@ -56,15 +56,18 @@ namespace vcpkg::Commands::Update
         }
         else
         {
+            std::string install_line;
             System::println("The following packages differ from their port versions:");
             for (auto&& package : outdated_packages)
             {
+                install_line += package.spec.to_string();
```
Comment (KindDragon) [inline toolsrc/src/commands_update.cpp]: What if `install_line` too big for a single line? Maybe add wrapping with indentation?

Comment (OlafvdSpek) [inline toolsrc/src/commands_update.cpp]: Then you buy a wider monitor. :D

Would wrapping play nice with copy/paste? How do we figure out what fits on a single line?

Comment (KindDragon) [inline toolsrc/src/commands_update.cpp]: Because of that they just print `<pkgs>...` :-)

Comment (KindDragon) [inline toolsrc/src/commands_update.cpp]: > How do we figure out what fits on a single line?

http://stackoverflow.com/a/12642749/61505 :-)

**Comment by ras0219-msft** [comment]
I think we can make some improvements here in the future (some sort of more intelligent indentation?), but I agree this is pretty much a strict improvement even if you do have a ton of libraries to update.

**Pull request merged**